chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,326 @@
+# Pre-trained models and datasets for text-to-speech
+
+Text-to-speech task (also called _speech synthesis_) comes with a range of challenges.
+
+First, just like in the previously discussed automatic speech recognition, the alignment between text and speech can be tricky.  
+The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
```

Comment:
```suggestion
However, unlike ASR, TTS is a one-to-many mapping problem, i.e. the same text can be synthesised in many different ways. Even different outputs (spectrograms or audio waveforms) can correspond to the same ground truth. The model has to learn to generate the correct duration and timing for each phoneme, word, or sentence which can be challenging, 
```

I think it is good to motivate the problem a bit more here. This helps the users center their expectations.